Source/swiftlint/Helpers/LintOrAnalyzeArguments.swift

```diff
@@ -20,11 +20,12 @@ struct LintOrAnalyzeArguments: ParsableArguments {
     var config = [String]()
     @Flag(name: [.long, .customLong("autocorrect")], help: "Correct violations whenever possible.")
     var fix = false
+
     @Flag(help: """
```

Comment:
I think there's a bug in `let_var_whitespace`, which is being triggered by the multiline `@Flag` - I'll open a defect for that once I have a good test case.

Reply:
See #4801